src/parser.js

```diff
@@ -356,7 +356,7 @@ export default class Parser {
                 ensureObject(node, 'raws');
                 node.raws.value = content;
 
-                spaceAfterMeaningfulToken = false;
+                spaceAfterMeaningfulToken = true;
```

Comment:
Not sure this is the right fix.

`spaceAfterMeaningfulToken` is used to detect a break in idents.

`[foo=bar i]` -> `i` is a separate token because the previous token was whitespace.

But a string token by definition means that the value has been seen.

`[foo="bar"i]` `i` is a separate token because the previous token was a string token.

Reply:
hm, why don't check we really have space?

Reply:
@alexander-akait thank you for questioning this.
A second look a day later lead me to the real issue.

https://github.com/postcss/postcss-selector-parser/pull/267/commits/03414790e7b91f2be826dc7aaeb51c7fa44e34c4